### ISSUE TITLE
[minLength] Parameter isn't a number, got parameter: true

### ISSUE THREAD

**Issue by martinszeltins** [victorgarciaesgi/regle#31]
I have a `minLength` rule but I'm getting an error and the rule isn't working. This was working in `0.2.11`

**Reproduction:**
[https://github.com/martinszeltins/regle-testing](https://github.com/martinszeltins/regle-testing)

````
[minLength] Parameter isn't a number, got parameter: true
Promise.then (async)
onUpdate:modelValue	@	index.vue:87
````

Here is my rule:
```ts
name: {
  required: applyIf(someCondition, required),
  minLength: applyIf(someCondition, minLength(3))
},
```

**Screenshot:**
![image](https://github.com/user-attachments/assets/92b333a9-926b-4fa2-b639-e95702c4f5fb)

**Comment by victorgarciaesgi** [comment]
Thanks! I need to add more tests to `applyIf`, didn't test this case.

**Comment by victorgarciaesgi** [comment]
Checked on your repo, fixed in `0.4.1`. Added test case for it ✅
